client/ionic/android/app/src/main/res/xml/config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+  <access origin="*" />
```

Comment:
A TODO and issue to review this.

Reply:
https://github.com/WorldHealthOrganization/app/issues/55

Reply:
TODO?

Reply:
(Yes, added in 57e153f, just failed to mention it)